### DISCUSSION TITLE
La nueva actualizacion 1.5.1 me esta dando infinidades de errores.

### DISCUSSION THREAD

**Discussion by Chuocloudstriker** [whooohq/whq-woocommerce-chilexpress-shipping#238] [Support]
Actualice a la version 1.5.1 junto con mi wordpress y mi woocomerce y me estan tando errores en el carrito en el checkout y no parece cumplir su funcion. que puede ser esto y donde consigo volver a la version anterior.

**Comment by EstebanForge** [comment]
@Cluodstriker estimado.
Entenderás que, entregando tan poco detalle de qué errores tienes, es imposible para nosotros hacer algo por ti al respecto.

Debes ser más claro respecto a qué error estás teniendo, como reproducirlo, etc. por favor.

Puedes ver el sitio de demo oficial de este plugin: https://wpchilexpress.whooohq.com/
En aquel sitio, está funcionando sin inconvenientes.

Detalles, por favor. Error. Pantallazos. Forma de reproducir el problema que tienes. Una URL donde ver en vivo el inconveniente. Etc.

Ten presente que es casi seguro que, si no usas el theme default de WC en tu instalación, te vamos a pedir realizar estos pasos: https://github.com/whooohq/whq-woocommerce-chilexpress-shipping#problemas-generales-visuales-glitches-yo-funcionales-en-el-carro-o-finalizar-compra-checkout y reportar como te fue con ello.

Quedo atento.

Esteban

**Reply by Chuocloudstriker** [reply]
Estos son los errores que me muestra en las pantallas de agregar al carrito, carrito de compra y checkout.
![image](https://user-images.githubusercontent.com/101353896/157878067-e7a925b6-6f88-48e3-b96d-576036edd6bf.png)

adicionalmente sin importar que direccion coloco en el checkout... no me da resultados de envio me dice que la zona no esta disponible.
![image](https://user-images.githubusercontent.com/101353896/157877981-0b6a96aa-a7ed-41ea-9199-cbaafaca2764.png)

**Comment by EstebanForge** [comment]
@Cluodstriker estimado, gracias por los pantallazos.

¿Puedes adjuntar un pantallazo de tu configuración del plugin, por favor?
En /wp-admin/admin.php?page=wc-settings&tab=shipping&section=chilexpress

Además, por favor instala este plugin https://wordpress.org/plugins/transients-manager/ y luego ir a Tools > Transients y presionar Delete All: https://i.imgur.com/bBntgAC.png

Luego vuelve a intentar obtener un precio en el checkout de tu tienda.

Probado recién en la tienda de demo, y el plugin está funcionando sin inconvenientes: https://i.imgur.com/qo7YILp.png

Esteban

**Reply by Chuocloudstriker** [reply]
Excelente durante el dia lo voy a intentar y te doy el Feedback. 👍 Gracias por el soporte.
